_posts/2024-10-14-Introduction-of-new-opensearch-website-search.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Introducing enhanced search functionality on OpenSearch.org
+authors:
+   - zelinhao
+   - kolchfa
+date: 2024-10-14 
+categories:
+  - community-updates
+  - update
+  - community
+meta_keywords: OpenSearch search update, improved search results, advanced filters, documentation search, OpenSearch search page, Amazon OpenSearch Service, version-specific search, search infrastructure update
+meta_description: Discover the new and improved search functionality on OpenSearch.org. Learn how our updated infrastructure, advanced filters, and dedicated search results page make it easier to find the documentation, blog posts, and event information you need.
+---
```

Comment:
Updated meta: 
meta_keywords: improve search functionality, OpenSearch search experience, OpenSearch.org, advanced filtering options, enhanced search experience

meta_description: Explore the enhanced search experience on OpenSearch.org. Discover how our upgraded infrastructure, advanced filtering options, and dedicated search page make it easier to locate the documentation and content you need.

Reply:
Updated.